app/models/concerns/foreman_openscap/model_extensions.rb

```diff
@@ -0,0 +1,8 @@
+module ForemanOpenscap
+  module ModelExtensions
+    extend ActiveSupport::Concern
+    included do
+      self.table_name_prefix = "foreman_openscap_"
```

Comment:
I think this should live in the engine. See [foreman_tasks engine](https://github.com/theforeman/foreman-tasks/blob/fd2206af4c9ce88ef08226ccac80a6d5224433ef/lib/foreman_tasks/engine.rb#L110-L112) for example